CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 ## Release History
 
+### 1.9.0-beta.1 (Unreleased)
+#### Breaking Changes
+* This change will stop updating the lease container continuation token based on kafka offset when using the config `connect.cosmos.offset.useLatest = true`, rather it will start the `ChangeFeedProcessor` from now processing changes from the current timestamp. [PR 516](https://github.com/microsoft/kafka-connect-cosmosdb/pull/516)
```

Comment:
```suggestion
* This change will stop updating the lease container continuation token based on kafka offset when using the config `connect.cosmos.offset.useLatest`. When set to `false`, connector will now process the changes from beginning.  When set to `true`, if the lease container exists, connector will now process the changes from the current continuation token in `leaseContainer`. If lease container doesn't exist, then it will start from beginning. If you want to start from beginning, it is advised to delete the lease container or change the kafka connector worker name. [PR 516](https://github.com/microsoft/kafka-connect-cosmosdb/pull/516)
```